.circleci/ci-docker-tag-op-geth-release.sh

```diff
@@ -27,12 +27,11 @@ fi
 echo "Tagging $SOURCE_IMAGE_TAG with '$IMAGE_TAG'"
 gcloud container images add-tag -q "$SOURCE_IMAGE_TAG" "$TARGET_IMAGE_TAG"
 
-# Do not tag with latest if the release is a release candidate.
-if [[ "$IMAGE_TAG" == *"rc"* ]]; then
-  echo "Not tagging with 'latest' because the release is a release candidate."
+# Only tag finalized releases with 'latest'
+if [[ "$IMAGE_TAG" =~ ^v1\.[0-9]+\.[0-9]+$ ]]; then
+  echo "Not tagging with 'latest' because the release is not finalized."
   exit 0
 fi
```

Comment:
Don't you want to negate the condition here? As is, won't we echo and exit for strings like `v1.101500.0` and continue for strings like `v1.101500.0-dev.1`

Reply:
indeed 🙈